ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml

```diff
@@ -1,14 +1,37 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
 binary_build_commands: make build
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-openshift-4.6:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6
+  to: projected-resource-csi-driver
```

Comment:
your output image name should be `csi-driver-projected-resource` to follow our existing conventions

Reply:
What does "projected resource" mean?

Reply:
> What does "projected resource" mean?

Please see https://github.com/openshift/enhancements/blob/master/enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md @smarterclayton 

Credit to @deads2k for coming up with the name.

Reply:
A broader overview from @adambkaplan on the overall work is at https://github.com/openshift/enhancements/blob/master/enhancements/subscription-content/subscription-content-access.md @smarterclayton

Reply:
@smarterclayton Everything has been renamed to csi-driver-projected-resource